Add instructions to set up dhcp server (#694)
aws/eks-anywhere: docs/content/en/docs/reference/vsphere/vsphere-dhcp.md

---
title: "Custom DHCP Configuration"
weight: 40
description: >
  Create a custom DHCP configuration for your vSphere deployment
---

If your vSphere deployment configred with DHCP and it cannot be configured with DHCP, you may want to run your own DHCP server.

> It may be necessary to turn off DHCP snooping on your switch to get DHCP working across VM main-servers.

Author: I'm not entirely sure either. I'll just change that to servers.

If you are running your administration machine in vSphere, it would most likely be easiest to run the DHCP server on that machine.
This example is are for Ubuntu.
 
# Install
Install DHCP server
```
sudo apt-get install isc-dhcp-server
```
 
# Configure /etc/dhcp/dhcpd.conf

Update the ip address range, subnet, mask, etc to suite your configuration similar to this:

```
default-lease-time 600;
max-lease-time 7200;
 
ddns-update-style none;
 
authoritative;
 
subnet 10.8.105.0 netmask 255.255.255.0 {
range 10.8.105.9  10.8.105.41;
option subnet-mask 255.255.255.0;
option routers 10.8.105.1;
 option domain-name-servers 147.149.1.69;
}
```
 
# Configure /etc/default/isc-dhcp-server

> Add the main interface to this file e.g: eth0 (this example uses ens160).

Reviewer: ```suggestion
Add the main NIC device interface to this file, such as eth0 (this example uses ens160).
```

 
```
INTERFACESv4="ens160"
```
 
# Restart DCHP

```

> service isc-dhcp-server restart

Reviewer: Is the service enabled by default? With Fedora, I think a separate step would be required to enable. If that is all that's needed in Ubuntu, ignore this comment.

Author: This looks correct to me. Installation seems to enable the service.
